packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -562,9 +564,8 @@ const GroupForm = ({
             </section>
           </section>
 
-          {(formType === 'create' || formType === 'edit') && (
-            <TopicGatingHelpMessage />
-          )}
+          {(formType === 'create' || formType === 'edit') &&
+            !allowlistEnabled && <TopicGatingHelpMessage />}
```

Comment:
is this check necessary here? you do this check inside `TopicGatingHelpMessage`

Reply:
We need this one here, otherwise TopicGatingHelpMessage will appear on both the members page as well as the /members/groups/create page, but we only want it to appear on one of the pages depending on the flag.